docs/source/examples/classification_and_regression.rst

```diff
@@ -0,0 +1,312 @@
+    print('The running processor is...', device) 
+
+CelebA Dataset
+==============
+
+We are going to use the CelebA dataset for this experiment. The CelebA dataset is a large-scale face attributes dataset which can be employed as the training and test sets for the following computer vision tasks: face attribute recognition, face detection, landmark (or facial part) localization, and face editing & synthesis.
+
+Fetching data 
+=============
+
+The section below consists of a few lines of codes that help us download the CelebA dataset from a public web source and unzip it. Downloading the CelebA dataset can be also done directly using `torch.datasets.CelebA(data_root, download=True)`. However, due to the high traffic on the dataset's Google Drive (the main source of the dataset), it usually fails to function. Hence we decided to download it from another public source but use it with `torch.datasets.CelebA()`.
+
+.. code-block:: python
+
+    data_root = 'datasets'
+    base_url = "https://graal.ift.ulaval.ca/public/celeba/"
+
+    file_list = [
+        "img_align_celeba.zip",
+        "list_attr_celeba.txt", 
+        "identity_CelebA.txt",
+        "list_bbox_celeba.txt",
+        "list_landmarks_align_celeba.txt",
+        "list_eval_partition.txt"
+    ]
+
+    # Path to folder with the dataset
+    dataset_folder = f'{data_root}/celeba'
+    os.makedirs(dataset_folder, exist_ok=True)
+
+    for file in file_list:
+        url = f"{base_url}/{file}"
+        if not os.path.exists(f"{dataset_folder}/{file}"):
+            wget.download(url, f"{dataset_folder}/{file}")
+
+    with zipfile.ZipFile(f"{dataset_folder}/img_align_celeba.zip", 'r') as ziphandler:
+        ziphandler.extractall(dataset_folder)
+   
+Now, as the dataset id downloaded, we can define our datasets and dataloaders in its original way.
+
+.. code-block:: python
+
+    transforms = tfms.Compose([
+        tfms.Resize((image_size, image_size)),
+        tfms.ToTensor(),
+        tfms.Normalize(imagenet_mean, imagenet_std)
+    ])
+    train_dataset = datasets.CelebA(
+        data_root,
+        split='train',
+        target_type=['attr', 'landmarks'],
+        transform=transforms
+    )
+    valid_dataset = datasets.CelebA(
+        data_root, 
+        split='valid', 
+        target_type=['attr', 'landmarks'], 
+        transform=transforms
+    )
+    test_dataset = datasets.CelebA(
+        data_root, 
+        split='test', 
+        target_type=['attr', 'landmarks'], 
+        transform=transforms
+    )
+                                
+    train_dataloader = DataLoader(train_dataset, batch_size=batch_size, shuffle=True)
+    valid_dataloader = DataLoader(valid_dataset, batch_size=batch_size, shuffle=False)
+    test_dataloader = DataLoader(test_dataset, batch_size=batch_size, shuffle=False)
+
+Here we can see how each dataset sample looks like:
+
+.. code-block:: python
+
+    print (train_dataset[0])
```

Comment:
An image of the output could be added here.

Reply:
<img width="809" alt="output" src="https://user-images.githubusercontent.com/50740990/128584636-a0572603-589e-443f-bd6c-b00323e75778.png">
 like this?

Reply:
Yeah, looks good.